src/util/setSelectionsAndFocusEditor.ts

```diff
@@ -78,21 +78,10 @@ function getViewColumn(editor: TextEditor): ViewColumn | undefined {
     return undefined;
   }
   const uri = editor.document.uri.toString();
-  const tabGroups: any[] = (window as any)?.tabGroups?.all ?? [];
-  const tabGroup = tabGroups.find(
-    (tabGroup: any) =>
-      !!tabGroup?.tabs.find(
-        (tab: any) => tab?.input?.modified?.toString() === uri
-      )
+  const tabGroup = (window as any)?.tabGroups?.all?.find((tabGroup: any) =>
+    tabGroup?.tabs.find((tab: any) => tab?.input?.modified?.toString() === uri)
```

Comment:
I think we prob want to support `original` and `uri` in addition to `modified`

Reply:
I decided to leave them out. Since original and modified is in the same tab focusing that tab would always focus the modified editor for some reason. I think it's a better behavior to throw an error than to focus the wrong editor. Including uri only gave us false positives like notebooks and I haven't found a use case where it's needed.